pychef/__init__.py

```diff
@@ -64,6 +82,45 @@
 )
 
 
+def interpret_images_to_doses_options(params, experiments):
+    """Interpret the dose.experiments options"""
+    dpi = params.dose.experiments.dose_per_image
+    if len(dpi) == 1:
+        doses_per_image = dpi * len(experiments)
+    elif len(dpi) != len(experiments):
+        raise ValueError(
+            """
+The dose_per_image option must provide either one value, or a number of values
+equal to the number of experiments (%s)"""
+            % len(experiments)
+        )
+    else:
+        doses_per_image = dpi
+
+    if params.dose.experiments.shared_crystal:
+        start_doses = [0]
+        accumulated_dose = 0
+        # adjust starting doses to account for a shared crystal.
+        for expt, dose_per_img in zip(experiments, doses_per_image):
```

Comment:
I think best to leave the experiments matching the input order (of data and dose_per_image) command to avoid confusion. This can be manually set anyway by giving starting doses.